spec/board_spec.rb

```diff
@@ -190,8 +198,25 @@
 
       expect(board.diag_12).to eq('....')
     end
-    
+  end
+
+  describe '#winner?' do
+    it 'returns false with no pieces played' do
+      board.create_layout
+      
+      expect(board.winner?).to be false
+    end
 
+    it 'returns true with 4 pieces in a _row_' do
```

Comment:
maybe change the "row" here to "in a line", since we're using row to describing horizontal as "row"

Reply:
I'm confused about this, we'll cover it later. I thought this was checking for a horizontal row.